salt/_states/metalk8s_package_manager.py

```diff
@@ -0,0 +1,55 @@
+"""Custom state for managing yum packages and dependencies"""
+
+from __future__ import absolute_import
+
+import logging
+
+from salt.exceptions import CommandExecutionError
+
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = "metalk8s_package_manager"
+
+
+def __virtualname__():
+    try:
+        os_family = __grains__['os_family'].lower()
+    except Exception:
+        return (False, "No yum based system detected")
```

Comment:
The above checks whether there's an `os_family` grain, which says nothing about being YUM based or not?

Reply:
my bad!